moveit_kinematics/kinematics_base_test/CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+cmake_minimum_required(VERSION 2.8.3)
```

Comment:
We can't have catkin package inside of a catkin package... moveit_kinematics is already a package

I think this subfolder should not be a ROS package but instead just a subfolder of moveit_kinematics